Gordon360/Controllers/ActivitiesController.cs

```diff
@@ -249,6 +249,7 @@ public ActionResult OpenSession(string id, string sess_cde)
         /// <returns></returns>
         [HttpPost]
         [Route("{involvement_code}/image")]
+        [AllowAnonymous]
```

Comment:
This needs to be removed since a user must be authenticated in order to post an image for an involvement.
```suggestion
```